skpro/distributions/multivariate_normal.py

```diff
@@ -0,0 +1,206 @@
+# copyright: skpro developers, BSD-3-Clause License (see LICENSE file)
+"""Multivariate Normal/Gaussian probability distribution."""
+
+__author__ = ["bhavikar04","fkiraly"]
+
+import numpy as np
+import pandas as pd
+from scipy.special import erf, erfinv
+from scipy.stats import multivariate_normal
+
+from skpro.distributions.base import BaseDistribution
+class Multivariate_Normal(BaseDistribution):
```

Comment:
It should be a space here. For that reason the pre-commit check is not passing

Reply:
also, class names should not contain underscores, they should be `UpperCamelCase`